shard-operator/src/main/resources/application.properties

```diff
@@ -42,5 +42,5 @@ quarkus.log.console.json.fields.sequence.enabled=false
 event-bridge.secret.name=addon-smart-events-operator-parameters
 # Name of the secret that needs to be imported on operator deployment.
 quarkus.kubernetes.env.secrets=${event-bridge.secret.name}
-
-
+event-bridge.istio.jwt.issuer=https://sso.redhat.com/auth/realms/redhat-external
+event-bridge.istio.jwt.jwksUri=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs
```

Comment:
I would use a param instead of hardcoded value like we do for
`event-bridge.sso.auth-server-url=${EVENT_BRIDGE_SSO_URL:http://${minikubeip}:30007/auth/realms/event-bridge-fm}`

Wdyt?

Reply:
done